Mail/mail.php

```diff
@@ -112,7 +112,7 @@ public function __construct($params = null)
      *               containing a descriptive error message on
      *               failure.
      */
-    public function send($recipients, $headers, $body)
+    public static function send($recipients, $headers, $body)
     {
         if (!is_array($headers)) {
             return PEAR::raiseError('$headers must be an array');
```

Comment:
_Non-static method PEAR::raiseError() should not be called statically_ would be caused by this call, and be based on the definition of the `PEAR::raiseError()` method, not this class's `send()` method.  If you upgrade your PEAR package version, that notice should go away.